cockroachdb/src/jepsen/cockroach/multiregister.clj

```diff
@@ -0,0 +1,131 @@
+(ns jepsen.cockroach.multiregister
+  "Multiple atomic registers test
+
+  Splits registers up into different tables to make sure they fall in
+  different ranges"
+  (:refer-clojure :exclude [test])
+  (:require [jepsen [cockroach :as cockroach]
+             [client :as client]
+             [checker :as checker]
+             [generator :as gen]
+             [reconnect :as rc]
+             [independent :as independent]
+             [util :as util]]
+            [jepsen.checker.timeline :as timeline]
+            [jepsen.cockroach.client :as c]
+            [jepsen.cockroach.nemesis :as cln]
+            [clojure.java.jdbc :as j]
+            [clojure.tools.logging :refer :all]
+            [knossos.model :as model]))
+
+(def reg-count 5)
+(def reg-range (range reg-count))
+
+(def table-prefix "String prepended to all table names." "register_")
```

Comment:
Do you want this to be different from the register test?

Reply:
It's fine if it differs. The register test should probably use a different table name but it doesn't really matter.